scripts/ensemble_forecast.py

```diff
@@ -54,11 +55,15 @@ def ensemble_forecast_batch(c, model_name):
 
         path = os.path.join(c.work_dir, 'cycle', t2s(c.time), model_name)
         output_dir = os.path.join(c.work_dir, 'cycle', t2s(c.next_time), model_name)
-        os.system("mkdir -p "+path)
+        os.makedirs(path, exist_ok=True)
+        os.makedirs(output_dir, exist_ok=True)
 
         job_opt = {'job_submit_cmd': c.job_submit_cmd,
+                   'job_submit_node': c.job_submit_node,
                    'path': path,
                    'time': c.time,
+                   'next_time': c.next_time,
+                   'n_ens': c.nens,
```

Comment:
I have passed c.nens in as 'nens' below at line 66, so you can just use kwargs['nens'] in the model.run code, no need for the additional n_ens. Well...I shall put the "'nens': c.nens" in job_opt, should be better readable.